src/bosh_azure_cpi/bin/test-unit

```diff
@@ -3,15 +3,12 @@
 args=$@
 [[ "$#" != "1" ]] && args=spec/unit/*
 
-pushd $SRC_DIR
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
SRC_DIR is defined in `.envrc` in the root directory of CPI. It needs `direnv`. I guess, in the CI, `direnv` is not installed. So, you replace it with `DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"`. Correct?

Reply:
yes do not want to make it depend on the direnv.